web/components/TopBar/ExportButton.vue

```diff
@@ -0,0 +1,67 @@
+<template>
+  <v-dialog v-model="data.dialog" width="500">
+    <template #activator="{ on }">
+      <v-btn color="primary ma-2" v-on="on"> Export </v-btn>
```

Comment:
nits: Let's change the color. 

```suggestion
      <v-btn color="ma-2" v-on="on"> Export </v-btn>
```